network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_reporter_page.html

```diff
@@ -212,8 +212,9 @@ <h4>{% trans "October 15, 2019" %}</h4>
         <h4>{% trans "October 26, 2019" %}</h4>
         <p>
           {% blocktrans with link="https://www.mozillafestival.org/speakers/2019/algorithmic-gaze/" trimmed %}
-          At MozFest 2019, Mozilla <a href="{{link}}" target="_blank">hosted</a> “The Algorithmic Gaze,” a panel exploring the dangers of untrustworthy AI like YouTube’s recommendation engine.</p>
+          At MozFest 2019, Mozilla <a href="{{link}}" target="_blank">hosted</a> “The Algorithmic Gaze,” a panel exploring the dangers of untrustworthy AI like YouTube’s recommendation engine.
         {% endblocktrans %}
+        </p>
```

Comment:
how did you even find this...

Reply:
It’s easier when you only look at what’s actually been extracted to .po files ;)